src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -328,22 +328,13 @@ public static async Task<OpenApiDocument> ConvertCsdlToOpenApi(Stream csdl, stri
             {
                 AddSingleQuotesForStringParameters = true,
                 AddEnumDescriptionExtension = true,
-                DeclarePathParametersOnPathItem = true,
                 EnableKeyAsSegment = true,
                 EnableOperationId = true,
-                ErrorResponsesAsDefault = false,
                 PrefixEntityTypeNameBeforeKey = true,
                 TagDepth = 2,
                 EnablePagination = true,
-                EnableDiscriminatorValue = true,
                 EnableDerivedTypesReferencesForRequestBody = false,
                 EnableDerivedTypesReferencesForResponses = false,
-                ShowRootPath = false,
```

Comment:
those settings are critical to Kiota based SDKs. If we want to move them somewhere else we first need to provide some kind of configuration experience on hidi's CLI and update the weekly generation pipeline in tandem. Otherwise we'll get a lot of side-effects.

Reply:
Kiota's configurations in the weekly generation pipeline have been saved in a file that will be consumed during generation.

Reply:
thanks for doing that. At this point why do we need to change anything in hidi anymore then?
Or if we change something, wouldn't it be better to remove any default from hidi and rely on the defaults from OpenApi.net.OData when no settings are provided?